grammars/ruby.cson

```diff
@@ -1940,6 +1952,10 @@
     'name': 'punctuation.separator.key-value'
   }
   {
+    'match': '->'
+    'name': 'keyword.operator.lambda.ruby'
```

Comment:
Yes - I agree it should be the same as `lambda`. Perhaps `lambda` is miscategorized, since it is part of core and not support? How about `storage.type.function.arrow` and change `lambda` to storage.type.function?